tap_instagram/streams.py

```diff
@@ -683,6 +693,9 @@ def parse_response(self, response: requests.Response) -> Iterable[dict]:
         if (
             resp_json.get("error", {}).get("error_user_title")
             == "Media posted before business account conversion"
+            or
+            str(resp_json.get("error", {}).get("message"))
+            in "(#10) Not enough viewers for the media to show insights"
```

Comment:
same comment about using `==`

Reply:
My understanding here is there is going to be additional text apart from the one I mentioned above. Here is the error I got when I faced the issue:

"400 Client Error: Bad Request - (#10) Not enough viewers for the media to show insights for path: /{media_id}/insights name=meltano run_id=5f77833c-280a-4b2e-adcd-97525b0ef76b"

In this case, "Bad Request - " would be present in the error message.